test/event-manager.spec.js

```diff
@@ -89,6 +89,11 @@ test('eventManager#destroy', t => {
     'WheelInput.destroy() should be called once');
   t.equal(keyInput.destroy.callCount, 1,
     'KeyInput.destroy() should be called once');
+
```

Comment:
Add a test that calls `destroy()` on the same EM twice? Sounds like that's what was happening in `react-map-gl` that we need to fail gracefully?